components/camel-as2/camel-as2-component/src/main/java/org/apache/camel/component/as2/AS2Configuration.java

```diff
@@ -258,6 +260,14 @@ public void setRequestUri(String requestUri) {
         this.requestUri = requestUri;
     }
 
+    public String getRequestUriPattern() {
+        return requestUriPattern;
+    }
+
+    public void setRequestUriPattern(String requestUriPattern) {
+        this.requestUriPattern = requestUriPattern;
+    }
```

Comment:
This will need to be documented, as it is part of the source that generated the component documentation.